.github/workflows/remote-copy.yml

```diff
@@ -194,15 +190,14 @@ jobs:
         # and then one locally on the runner to copy the list of files from the remote.
         run: |
           ssh ${{ secrets.SSH_USER }}@${{ secrets.SSH_HOST }} -i ${{ steps.ssh.outputs.private-key-path }} /bin/bash <<'EOT'
-
           # Create intermediate directories since we can't use `rsync`s `--mkpath` on some deployment targets
-          if [ -d "${{ inputs.source }}" ]; then
-            mkdir -p "${{ inputs.target }}"
-          else  # we can assume we are transferring a file
-            mkdir -p $(dirname "${{ inputs.target }}")
-          fi
+          mkdir -p "${{ inputs.target }}"
+
+          # The output of the rsync command will be a list of transferred absolute paths, for further processing
+          RSYNC_OUT_FORMAT="$(readlink -f "${{ inputs.target }}")/%n"
```

Comment:
You've got nested double quotes. Did you mean to just evaluate `$(readlink -f "${{ inputs.target }})` and concatenate `/%n`, or is this supposed to be evaluated dynamically by `rsync`?

Reply:
Nested double quotes are okay if they're in `$(...)`, I believe. Verified:
```bash
RSYNC_OUT_FORMAT="$(readlink -f "/opt/")/%n"
echo $RSYNC_OUT_FORMAT
/opt/%n
```

Reply:
Did not know that. Thanks.